### PR TITLE
Returning bytesRead per query as part of searchResult

### PR DESCRIPTION
**Pull request by Thejas-bhat**

### CODE REVIEW

File: read.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func (s *SegmentBase) getDocStoredMetaAndCompressed(docNum uint64) ([]byte, []by
 	meta := s.mem[storedOffset+n : storedOffset+n+metaLen]
 	data := s.mem[storedOffset+n+metaLen : storedOffset+n+metaLen+dataLen]
 
-	s.incrementBytesRead(metaLen + dataLen)
+	// s.incrementBytesRead(metaLen + dataLen)
```
Comment (abhinavdangeti) [inline read.go:25]: Why the commented code?

File: read.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func (s *SegmentBase) getDocStoredOffsets(docNum uint64) (
 	dataLen, read := binary.Uvarint(s.mem[storedOffset+n : storedOffset+n+binary.MaxVarintLen64])
 	n += uint64(read)
 
-	s.incrementBytesRead(n)
+	// s.incrementBytesRead(n)
```
Comment (abhinavdangeti) [inline read.go:44]: Ditto.

File: read.go (hunk 1 above)
Comment (Thejas-bhat) [reply read.go]: Yeah resolving them, my bad.

**Comment by abhinavdangeti** [comment]
@Thejas-bhat Use bleve_index_api@v1.0.5 and scorch_segment_api@v2.1.4.

**Review by abhinavdangeti — CHANGES_REQUESTED** [review]
It seems this PR is incomplete.

**Review by abhinavdangeti — APPROVED** [review]

**Pull request merged**